xla/service/gpu/cudnn_fusion_test.cc

```diff
@@ -0,0 +1,218 @@
+/* Copyright 2024 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Can you please move this file to fusions/?

Reply:
Done.